guides/dai_transaction_from_browser.md

```diff
@@ -0,0 +1,65 @@
+# Transacting DAI on OmiseGO Network through the browser
+
+Estimated time: 5 minutes
+
+Goal:
+
+- To understand plasma transactions with ERC20 token using DAI
+- To get first hand experience in making ERC20 interactions with plasma
+- To learn the process of making `deposit`, `transact` and `exit` with ERC20
+
+In this guide, we will introduce to you the Plasma Interface by making transactions directly from your browser. Just make sure you have the following installed:
+1. node-js >10.0.0
+2. npm > 6.0.0
+
+Have access to a code editor, browser with a web3 wallet like MetaMask
+
+### Note
+
+This guide is meant for **Samrong** and any v0.2 instance of OMG Network
+
+### Initial Setup
+1. Clone the latest version of `js-starter-kit` onto your local machine. The repo can be found here https://github.com/omisego/js-starter-kit
+
+2. Open up the `config.js` on your text editor
+
+3. Replace the current configs in `config.js` with the  [following Network Endpoints for Samrong](https://github.com/omisego/dev-portal/blob/master/guides/network_endpoints.md) then save the file.
+
+4. install all the dependencies `npm install`
+
+5. start the app up by running `npm run dev`
+
```

Comment:
### Getting started
This section explains what you'll need to prepare for the initial setup. 

1. Clone the latest version of `js-starter-kit` onto your local machine. 
    You can find the repo here: https://github.com/omisego/js-starter-kit

 2. In a text editor, open up the following file: `config.js`

 3. In `config.js`, replace the current configuration with the network endpoints for Samrong; then, save your changes. 
 You'll find the network endpoints for Samrong in this file:  https://github.com/omisego/dev-portal/blob/master/guides/network_endpoints.md

 4. Run `npm install' to install all the dependencies.

 5. Run `npm run dev` to start the app.